src/Uno.UWP/UI/Input/GestureRecognizer.Manipulation.cs

```diff
@@ -269,6 +269,10 @@ private ManipulationDelta GetCumulative()
 			{
 				var translateX = _isTranslateXEnabled ? _currents.Center.X - _origins.Center.X : 0;
 				var translateY = _isTranslateYEnabled ? _currents.Center.Y - _origins.Center.Y : 0;
+#if __MACOS__
+				// correction for translateY being inverted (#4700)
+				translateY *= -1;
```

Comment:
I'm really surprised by this one ... this would mean that coordinates in all pointer events are invalid on MacOS. Did you validate that we have the right Y when doing `PointerRoutedEventArgs.GetCurrentPoint()` ?

Also if this is required in `GetCumulative()` it's also probably needed for `GetDelta()`

Reply:
`GetDelta()` seems to be solely based on the result of `GetCumulative()`. Assuming the source values are now correct, the aggregated value _should also_ be correct.